Common/ppcEmitter.h

```diff
@@ -275,8 +275,12 @@ namespace PpcGen
 		// 32 bit
 		void LWZ	(PPCReg dest, PPCReg src, int offset = 0);
 		void LWBRX	(PPCReg dest, PPCReg src, PPCReg offset);
+#if PPSSPP_ARCH(64BIT)
 		// 64 bit
 		void LD		(PPCReg dest, PPCReg src, int offset = 0);
+#else
+		void LD		(PPCReg dest, PPCReg src, int offset = 0) { if(!src) Crash(); LWZ(dest, src, offset); }
```

Comment:
Should LD be supported at all if 32-bit PPC? It's only a valid instruction in 64-bit mode, right, so caller should make sure to avoid it?

Reply:
well it was either this or putting ifdefs everywhere where register state is being saved/restored.
LD is only being used for that purpose. although you are right that it can end up badly if a 64-bit load is actually needed. not sure how to proceed here...

Reply:
maybe add a psoeudo instruction for that purpose ?

Reply:
ok looks like LD and STD aren't being used that often afterall.
I'll change this to ifdef's at the call site instead.

Reply:
but... surely if it was previously actually loading only 32 out of 64-bit, it should always have been using a 32-bit load, even on PPC64. If it were actually loading all 64 bits, you'll need different code now anyway?  I guess except when loading pointers that the JIT uses for its own purposes maybe.. for that kind of purpose a pseudo instruction would be nice.

Reply:
it was only being used to save/restore the callee saved registers in the prologue/epilogue, which is why it only needed a 32bit load/store in 32bit mode.

Reply:
Ah I see. Yeah would definitely prefer ifdefs at the call site then.